requirements/production.txt

```diff
@@ -2,7 +2,7 @@
 bleach>=1.5.0,<3.1
 channels>=2.1.2,<2.2
 daphne>=2.2,<2.3
-Django>=1.11,<2.2
+Django>=2.1.5,<2.2
```

Comment:
I also thought about this but I suggest, that we don't fix the django version.

Your like will unsupport all old django versions without any technical need. OpenSlides runs fine on Django 1.11 and Django 2.0 on the latest version.

We could write a more complicated version identifier, that supports older versions of django but not 2.1.4, but then it is harder to automatically check for the support for old versions. I do this constantly with `python make min_requirements`.

And OpenSlides can be run with django 2.1.4. You only need sqlite 3.25.

So in conclusion I would suggest, that we just leave this as it is. If you like, we can write in the readmy, that we only support the latest point release of any dependency. Other projects do the same. See: https://docs.djangoproject.com/en/2.1/faq/install/#faq-python-version-support

Reply:
To clarify:
Each django version <=2.1.4 failed with current stable sqlite 3.26.x!
Because sqlite 3.26 is an important security release which fixed a critical vulnerability almost each distribution has updated sqlite to current version. These users has to upgrade to django>=2.1.5 to run OpenSlides.
#4082 shows that other users alread run into problems when we don't update our requirements.

So I prefer to cut support for old django versions - with a comment in Changelog (that django>=1.11,<2.1.5 works with sqlite 3.25.x).
@ostcar ok?

Reply:
That is not true. You can test it easily with ` pip install "Django<2.1"`. This runs without any problem. The reason is, that django still supports the versoins 1.11 and 2.0. They backported the fix to this versions. See https://docs.djangoproject.com/en/2.1/releases/2.0.10/

Reply:
You are right! I tested only the older 2.0 release.
The three new django versions 2.1.5, 2.0.10, and 1.11.18 supports the new sqlite 3.26 version. And yes, I quite agree with you that an exclusion of all other django versions in requirements.txt will be to complicated. Each user has to run `pip install -U -r requirements.txt` or `pip install -U openslides` to get the latest working django version. This should enough. 

I'll close this pull request.